Course.java

```diff
@@ -0,0 +1,47 @@
+public class Course {
+  private String courseName;
+  public int classSize =0;
```

Comment:
This should probably be protected, otherwise I can do `course.classSize = 1` and override the behavior.